lib/spack/spack/cmd/uninstall.py

```diff
@@ -338,6 +335,22 @@ def uninstall_specs(args, specs):
     do_uninstall(env, uninstall_list, args.force)
 
 
+def confirmation_before_removal(specs):
+    """Displays the list of specs that are candidate for removal and asks
```

Comment:
I don't have a preference other these things, so will do. I have to note though that this is confusing, since I thought we were trying to adopt [Google Style docstrings](http://google.github.io/styleguide/pyguide.html#383-functions-and-methods) which say:

> The docstring should be descriptive-style ("""Fetches rows from a Bigtable.""") rather than imperative-style ("""Fetch rows from a Bigtable."""), except for property data descriptors, which should use the same style as attributes.

Reply:
Done